File: fetch2/src/main/java/com/tonyodev/fetch2/FetchDatabase.java
package com.tonyodev.fetch2;

import android.arch.persistence.room.RoomDatabase;
import android.arch.persistence.room.Database;

/**
 * Created by tonyofrancis on 6/14/17.
 */

@Database(entities = {RequestInfo.class},version = 1,exportSchema = false)
public abstract class FetchDatabase extends RoomDatabase {
    public abstract RequestInfoDao requestInfoDao();
}

File: fetch2/src/main/java/com/tonyodev/fetch2/FetchDatabase.kt
package com.tonyodev.fetch2

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

/**
 * Created by tonyofrancis on 6/14/17.
 */

@Database(entities = arrayOf(RequestInfo::class), version = 1, exportSchema = false)
abstract class FetchDatabase : RoomDatabase() {
    abstract fun requestInfoDao(): RequestInfoDao
}
